guides/join_space_mutation.md

```diff
@@ -0,0 +1,49 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'join-space-mutation'
+is: 'experimental'
+---
+
+# Join Space
+
+## Concepts
+
+The _joinSpace_ mutation allows the user to join a space. Currently this is only applicable to spaces the user is invited to already. The mutation accepts a Space ID as an argument and makes the request on behalf of the calling user.
```

Comment:
"Currently this is only applicable to spaces to which the user has already been invited."